wordpress-mobile/WordPress-Android
# Fix dark mode text contrast issue

Fixes #
### Before Fix
![bad-dark](https://github.com/wordpress-mobile/WordPress-Android/assets/27432070/37766261-9265-4877-877e-ae03fb2f471f)

### After Fix
![fix-dark](https://github.com/wordpress-mobile/WordPress-Android/assets/27432070/fa92254f-1fcf-4f18-969f-99b35045da2c)

To test:

1. Launch the JP app on an Android device.
2. Make sure the app is in dark mode. 
3. Navigate to the "Notifications" tab.
4. Make sure that the warning message "Push notifications are turned off" is well visible in dark mode, as shown in the screenshots.

## Regression Notes
1. Potential unintended areas of impact

5. What I did to test those areas of impact (or what existing automated tests I relied on)
NaN

6. What automated tests I added (or what prevented me from doing so)
NaN
PR submission checklist:


UI Changes testing checklist:
